### DISCUSSION TITLE
example scenes - cant find textures + references

### DISCUSSION THREAD

**Discussion by nospitters** [dreamworksanimation/openmoonray#57] [Q&A]
hi,

first thx a lot in providing the moonray renderer as open source...

i am more a hobbiest-coder, and i successfully compiled on centos 7 VMWare (after a few hours of troubleshooting...).
the rectangle testscene renders fine and i wanted to do some more with the example scenes...
but when rendering i.e. the "bathroom" i get errors that it cant find the textures and some other references, even they are there in the "textures" folder, and so the render just renders a white plane (assume this is the window)...
the scene.rlda contains relative pathes to the textures and other mesh references, right (./textures/xyz.tx)...
any hints?

**Comment by gonsolo** [comment]
You have to start the rendering from the bathroom folder as the texture references are local paths. Also you have to include both scenes (-in scene.rdla -in scene.rdlb) as the geometry is in the binary file. I was successful after trying a bit: https://github.com/dreamworksanimation/openmoonray/issues/45#issuecomment-1475182224
